concepticondata/concepticon.tsv

```diff
@@ -3449,3 +3449,10 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3463	RIVERBANK	The physical world	The margin or shore of a river.	Person/Thing	
 3464	SHORE OF LAKE	The physical world	The margin or shore of a lake.	Person/Thing	
 3465	SHORE OF SEA	The physical world	The margin or shore of a sea or ocean.	Person/Thing	
+3466	DANGEROUS	Emotions and values	Capable of injuring or harming someone.	Property	
+3467	HELP (AID)	Social and political relations	The act of helping someone.	Person/Thing	
+3468	KINDNESS	Emotions and values	The quality of being caring, generous and helpful towards other people.	Property	
+3469	LEVEL	Quantity	The amount or degree of something; the height of the surface of something liquid in respect to its bottom.	Person/Thing	
+3470	LIAR	Social and political relations	The person who tells lies.	Person/Thing	
```

Comment:
`The person` -> `A person`. These constructions require indefinite article usually.